trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.test.ts

```diff
@@ -264,20 +280,22 @@ describe('controllers/utilisation-reports/utilisation-report-reconciliation-for-
         originalUrl,
       });
 
+      const feeRecordPaymentGroups = [
+        {
+          ...aFeeRecordPaymentGroup(),
+          paymentsReceived: null,
+          totalPaymentsReceived: null,
+        },
+        {
+          ...aFeeRecordPaymentGroup(),
+          paymentsReceived: null,
+          totalPaymentsReceived: null,
```

Comment:
These objects are repeated 5x times in this test, could we create a single DRY fixture, maybe `emptyFeeRecordPaymentGroup`?

Reply:
How about `minimalFeeRecordPaymentGroup`?

Reply:
I think if I read that without the context of this conversation I would be confused and would need to click through to the definition and read it to understand the test, so I think for clarity's sake we should either leave it as the original, or be DRY but with an explicit name so as not to obscure anything.